src-electron/main-process/modules/wallet-rpc.js

```diff
@@ -1218,16 +1269,20 @@ export class WalletRPC {
       let sweep_all = amount == this.wallet_state.unlocked_balance;
 
       const rpc_endpoint = sweep_all ? "sweep_all" : "transfer_split";
-      const params = sweep_all
+      const rpcSpecificParams = sweep_all
```

Comment:
```suggestion
      const sendOptions = sweep_all
```

Reply:
yeah, thought for a bit on a name for this var, reason I named it that is because it's specific to the particular rpc being called, where something like `sendOptions` doesn't make that as clear

Reply:
Ah maybe `transferParams` then?

Reply:
if a sweep is done, then they will be the sweep params

Reply:
Alright, we'll just keep this until we can think of something better 👍